src/storage/btree.c

```diff
@@ -24826,7 +24827,7 @@ btree_range_scan_descending_fix_prev_leaf (THREAD_ENTRY * thread_p, BTREE_SCAN *
   VPID_COPY (&prev_leaf_vpid, next_vpid);
 
   /* Conditional latch for previous page. */
-  prev_leaf = pgbuf_fix (thread_p, &prev_leaf_vpid, OLD_PAGE, PGBUF_LATCH_READ, PGBUF_CONDITIONAL_LATCH);
+  prev_leaf = pgbuf_fix_old_and_check_repl_desync (thread_p, prev_leaf_vpid, PGBUF_LATCH_READ, PGBUF_CONDITIONAL_LATCH);
```

Comment:
below, after
```
  /* Conditional latch failed. */
```
add a comment explaining that the page can be null also because of page desync on PTS and that the behavior is the same as in other previouslty existing situations.